test/util.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2016 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package test
+
+import (
+	"bytes"
+	"encoding/json"
+)
+
+func AreEqualJSON(s1, s2 string) (bool, error) {
```

Comment:
I'd rather a `==` or `bytes.Equal()` as well.  While white space isn't part of the spec, a change there should be investigated.

If we are going to keep `AreEqualJSON` though, I'd rather the name be changed to `JSONEqual`.